events/2020-online-get-together.md

```diff
@@ -59,8 +59,10 @@ Wednesday, 2 December:
 
 ### Proposal submissions
 
-[Abstract submission form](https://indico.neic.no/event/146/)
-is open. Submission deadline: 16th November 2020, 23:59 CET.
+[Abstract submission form](https://indico.neic.no/event/146/) is
+open. Submission deadline: 16th November 2020, 23:59 CET (but
+proposals will continue to be accept on a rolling, best-effort basis
+after this - please submit as soon as you can, even if not perfect).
```

Comment:
Sounds a little bit desperate IMO, maybe something like "we encourage you to submit..." sounds better.

Reply:
I agree, and went ahead and changed the wording.